src/createApp.ts

```diff
@@ -512,6 +513,20 @@ function addIdentifier(app: App, id: Identifier) {
 	};
 }
 
+function createCustomWidget(app: App, id: string) {
+	const customFactories = customElementFactories.get(app);
+
+	return app.defaultWidgetStore.get(id).then((state: any) => {
```

Comment:
`customFactory.get` will throw if a factory for the type doesn't exist, so the `customFactory` should never be undefined.